build_local_windows.ps1

```diff
@@ -53,6 +51,42 @@ python -m venv "$tmpdir/dcos_build_venv"
 
 # Install the DC/OS tools
 ./prep_local_windows.ps1
+# Build a release of DC/OS
+release create $env:USERNAME local_build windows
+
+##Write DCOS installer locally
+$config_yaml =
+"storage: `
+   local: `
+    kind: local_path `
+    path: $myhome/dcos-artifacts `
+options: `
+  preferred: local `
+  cloudformation_s3_url: https://s3-us-west-2.amazonaws.com/downloads.dcos.io/dcos"
+
+   $config_yaml | Set-Content -Path "dcos-release.config.yaml"
 
+type dcos-release.config.yaml
+# Create a python virtual environment to install the DC/OS tools to
+python -m venv "$tmpdir/dcos_build_venv"
+. "$tmpdir/dcos_build_venv/Scripts/Activate.ps1"
+
+# Install the DC/OS tools
+./prep_local_windows.ps1
 # Build a release of DC/OS
 release create $env:USERNAME local_build windows
+
+# Build tar ball for windows. 2 params: packages location and DC/OS variant:
+./build_genconf_windows.ps1 "$HOME\dcos-artifacts\testing"
+
+##Import AWS modules on Azure TeamCity runner
+Install-Module -Name AWS.Tools.Common -Force
+Install-Module -Name AWS.Tools.S3 -Force
+
+## Set and Read AWS Credentials:
+Set-AWSCredential -AccessKey $env:AWS_ACCESS_KEY_ID -SecretKey $env:AWS_SECRET_ACCESS_KEY -StoreAs aws_s3_windows
+Set-AWSCredential -ProfileName aws_s3_windows
+## Upload Tar Ball to dcos.download.io
+Write-S3Object -BucketName "s3-us-west-2" -Key "downloads.dcos.io\dcos\windows\$env:BRANCH\commit\$env:COMMIT\dcos_generate_config_win.sh" -File "$HOME\dcos-artifacts\windows\dcos_generate_config_win.sh" -CannedACLName public-read
```

Comment:
Is this correct? - Sorry the s3-us-west-2 is actually the CDN URI.

The bucket name is **downloads.dcos.io**


The rest is the directory name.

Reply:
![2019-10-31_09-10](https://user-images.githubusercontent.com/332330/67965026-837ee400-fbbe-11e9-90ab-bb7baa7db4df.png)

Reply:
![2019-10-31_09-10_2](https://user-images.githubusercontent.com/332330/67965063-92fe2d00-fbbe-11e9-8023-e442f842ac91.png)
![2019-10-31_09-10_1](https://user-images.githubusercontent.com/332330/67965064-9396c380-fbbe-11e9-9a18-38555f47975e.png)

Reply:
Please notice the directory structure.